src/System.IO.FileSystem/tests/Directory/GetFileSystemEntries_str_str.cs

```diff
@@ -644,6 +643,9 @@ public void WindowsSearchPatternInvalid_Core()
                     case '/':
                         Assert.Throws<DirectoryNotFoundException>(() => GetEntries(Directory.GetCurrentDirectory(), string.Format("te{0}st", invalidChar.ToString())));
```

Comment:
This is the only case that might be different between Unix/Windows. We can use platform detection if needed.

Reply:
Yes but this is the test called `WindowsSearchPatternInvalid_Core` which runs in `TestPlatforms.Windows` only. So if I am under the right impression, then this case should stay as is.

Let me check if there's a similar test but for Unix. If the test case for `'/'` does not exist, I'll add it.

Reply:
I read your comment below. I'll make this test x-platform and will do different checks for `'/'` depending on the platform.